terraform-provider-intro/order-create.md

```diff
@@ -0,0 +1,55 @@
+Now that you have initialized your Terraform workspace, you will create the order using Terraform.
+
+Add the following to your `main.tf`{{open}} file.
+
+<pre class="file" data-filename="main.tf" data-target="append">
+provider "hashicups" {
+  username = "education"
+  password = "test123"
+}
+
+resource "hashicups_order" "edu" {
+  items {
+    coffee {
+      id = 3
+    }
+    quantity = 2
+  }
+  items {
+    coffee {
+      id = 2
+    }
+    quantity = 2
+  }
+}
+
+output "edu_order" {
+  value = hashicups_order.edu
+}
+</pre>
+
+Run `terraform apply` to create the order. Notice how the execution plan shows a proposed order, with additional information about the order.
+
+`terraform apply`{{execute T2}}
+
+Remember to confirm the apply step with a `yes`{{execute T2}}.
```

Comment:
```suggestion
Notice how the execution plan shows a proposed order, with additional information about the order.

Remember to confirm the apply step with a `yes`{{execute T2}}.
```

Reply:
Also when testing scenarios, occasionally people accidentally ran the `yes` command (when they clicked on the yes bit before the `terraform apply` bit. I worked around this by adding an alias to coursebase.sh: https://github.com/hashicorp/katakoda/blob/9426a67462da92cb2b7b2120857347bfc4c1c297/terraform-organize-configuration-workshop/courseBase.sh#L9